.github/workflows/main.yml

```
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [10.x, 12.x, 14.x, 16.x]
```

`package.json` requires >= node 12.

```suggestion
        node-version: [12.x, 14.x, 16.x]
```